### PR TITLE
Added source capability

### PR DESCRIPTION
**Pull request by agarneha1331**

### CODE REVIEW

**Review by karanshah-browserstack — DISMISSED** [review]

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]
Can make it Sel 4 compliant

File: src/main/java/com/browserstack/demos/BrowserStack.java [hunk 1]
```diff
@@ -2,17 +2,31 @@
...
+        bstackOptions.put("osVersion", "7");
+        bstackOptions.put("sessionName", "BStack automate-java");
+        bstackOptions.put("buildName", "browserstack-build-1");
+        bstackOptions.put("source", "automate-java:sample-selenium-4:v1.0");
+        bstackOptions.put("projectName", "My Awesome App");
+        bstackOptions.put("debug", true);
+        bstackOptions.put("seleniumVersion", "3.12.0");
+
+        MutableCapabilities capabilities = new MutableCapabilities();
+        capabilities.setCapability("browserstack.use_w3c", true);
```
Comment (kamal-kaur04) [inline src/main/java/com/browserstack/demos/BrowserStack.java]: remove this

File: src/main/java/com/browserstack/demos/BrowserStack.java [hunk 2]
```diff
@@ -2,17 +2,31 @@
...
+        capabilities.setCapability("browserstack.browserName", "Chrome");
```
Comment (kamal-kaur04) [inline src/main/java/com/browserstack/demos/BrowserStack.java]: `browserName` is the cap

File: src/main/java/com/browserstack/demos/BrowserStack.java [hunk 3]
```diff
@@ -2,17 +2,31 @@
...
-        ChromeOptions options = new ChromeOptions();
+        HashMap<String, Object> bstackOptions = new HashMap<String, Object>();
+        bstackOptions.put("os", "Windows");
+        bstackOptions.put("osVersion", "7");
+        bstackOptions.put("sessionName", "BStack automate-java");
+        bstackOptions.put("buildName", "browserstack-build-1");
+        bstackOptions.put("source", "automate-java:sample-selenium-4:v1.0");
+        bstackOptions.put("projectName", "My Awesome App");
+        bstackOptions.put("debug", true);
+        bstackOptions.put("seleniumVersion", "3.12.0");
```
Comment (kamal-kaur04) [inline src/main/java/com/browserstack/demos/BrowserStack.java]: remove seleniumVersion

File: src/main/java/com/browserstack/demos/BrowserStack.java [hunk 4]
```diff
@@ -2,17 +2,31 @@
...
 public class BrowserStack {
 
     public static void main(String args[]) throws MalformedURLException {
-        ChromeOptions options = new ChromeOptions();
+        HashMap<String, Object> bstackOptions = new HashMap<String, Object>();
+        bstackOptions.put("os", "Windows");
+        bstackOptions.put("osVersion", "7");
+        bstackOptions.put("sessionName", "BStack automate-java");
+        bstackOptions.put("buildName", "browserstack-build-1");
+        bstackOptions.put("source", "automate-java:sample-selenium-4:v1.0");
```
Comment (kamal-kaur04) [inline src/main/java/com/browserstack/demos/BrowserStack.java]: modify source key

File: src/main/java/com/browserstack/demos/BrowserStack.java [hunk 5]
```diff
@@ -2,17 +2,31 @@
...
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.MutableCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
 public class BrowserStack {
 
     public static void main(String args[]) throws MalformedURLException {
-        ChromeOptions options = new ChromeOptions();
+        HashMap<String, Object> bstackOptions = new HashMap<String, Object>();
```
Comment (kamal-kaur04) [inline src/main/java/com/browserstack/demos/BrowserStack.java]: `<String, String>`

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: src/main/java/com/browserstack/demos/BrowserStack.java (hunk 1 above)
Comment (agarneha1331) [reply src/main/java/com/browserstack/demos/BrowserStack.java]: capabilities.setCapability("browserstack.use_w3c", true);
This one??

Comment (kamal-kaur04) [reply src/main/java/com/browserstack/demos/BrowserStack.java]: yes

**Review by kamal-kaur04 — APPROVED** [review]

**Review by francisf — APPROVED** [review]

**Pull request merged**